### PR TITLE
GH-710: fixing TTHxxxArgs 

### PR DESCRIPTION
**Pull request by bertllll**

### CODE REVIEW

File: node/src/proxy_server/mod.rs [hunk 1]
```diff
@@ -2415,19 +2418,17 @@ mod tests {
...
-                source_addr,
-                timestamp: now,
-                is_decentralized: true,
-            },
-            logger: &logger,
-            hopper_sub: &peer_actors.hopper.from_hopper_client,
-            dispatcher_sub: &peer_actors.dispatcher.from_dispatcher_client,
-            accountant_sub: &peer_actors.accountant.report_services_consumed,
-            add_return_route_sub: &peer_actors.proxy_server.add_return_route,
+        let local_tth_args = TryTransmitToHopperArgs {
```
Comment (dnwiebe) [inline node/src/proxy_server/mod.rs]: You didn't like this variable name.

File: node/src/proxy_server/mod.rs [hunk 2]
```diff
@@ -2501,20 +2502,18 @@ mod tests {
...
-                timestamp: SystemTime::now(),
-                is_decentralized: false,
-            },
-            logger: &logger,
-            hopper_sub: &peer_actors.hopper.from_hopper_client,
-            dispatcher_sub: &peer_actors.dispatcher.from_dispatcher_client,
-            accountant_sub: &peer_actors.accountant.report_services_consumed,
-            add_return_route_sub: &peer_actors.proxy_server.add_return_route,
-            retire_stream_key_sub_opt: Some(&peer_actors.proxy_server.stream_shutdown_sub),
+        let local_tth_args = TryTransmitToHopperArgs {
```
Comment (dnwiebe) [inline node/src/proxy_server/mod.rs]: Change this one too?

File: node/src/proxy_server/mod.rs [hunk 3]
```diff
@@ -2753,19 +2752,17 @@ mod tests {
...
-                source_addr,
-                timestamp: SystemTime::now(),
-                is_decentralized: true,
-            },
-            logger: &logger,
-            hopper_sub: &peer_actors.hopper.from_hopper_client,
-            dispatcher_sub: &peer_actors.dispatcher.from_dispatcher_client,
-            accountant_sub: &peer_actors.accountant.report_services_consumed,
-            add_return_route_sub: &peer_actors.proxy_server.add_return_route,
+        let local_tth_args = TryTransmitToHopperArgs {
```
Comment (dnwiebe) [inline node/src/proxy_server/mod.rs]: And here?

File: node/src/proxy_server/mod.rs [hunk 4]
```diff
@@ -4851,12 +4848,17 @@ mod tests {
...
     fn resolve_route_query_response_handles_error() {
         init_test_logging();
+        let cryptde = main_cryptde();
         let recorder = Recorder::new();
         let addr = recorder.start();
         let add_route_msg_sub = recipient!(&addr, AddRouteMessage);
         let logger = Logger::new("resolve_route_query_response_handles_error");
-        let movable_tth_args = TTHMovableArgs {
-            common_opt: None,
+        let movable_tth_args = TryTransmitToHopperArgs {
```
Comment (dnwiebe) [inline node/src/proxy_server/mod.rs]: One more.

**Review by dnwiebe — APPROVED** [review]

**Review by utkarshg6 — APPROVED** [review]
LGTM! 👍

**Pull request merged**